idp/openid/openid-connect.go

```diff
@@ -47,6 +47,25 @@ func init() {
 	})
 }
 
+// An IdentityCreator is used to create a candid identity from the
```

Comment:
nitpick: s/An IdentityCreator/IdentityCreator/

Reply:
The convention for documenting types is to start the comment with an indefinite article (e.g. https://golang.org/pkg/net/http/#Request)